ice/server.py

```diff
@@ -0,0 +1,8 @@
+from uvicorn import main
```

Comment:
Why main instead of run?

https://www.uvicorn.org/deployment/#running-programmatically

Reply:
This allows reusing all the `click` options, e.g. `python -m ice.server --port ... --host ...`